packages/@expo/fingerprint/src/sourcer/SourceSkips.ts

```diff
@@ -2,35 +2,54 @@
  * Bitmask of values that can be used to skip certain parts of the sourcers when generating a fingerprint.
  */
 export enum SourceSkips {
-  // Skip nothing
+  /** Skip nothing */
   None = 0,
 
-  // Versions in app.json, including Android versionCode and iOS buildNumber
+  //#region - ExpoConfig source (e.g., app.json, app.config.js, etc.)
+
+  /** Versions in app.json, including Android versionCode and iOS buildNumber */
   ExpoConfigVersions = 1 << 0,
 
-  // runtimeVersion in app.json if it is a string
+  /** runtimeVersion in app.json if it is a string */
   ExpoConfigRuntimeVersionIfString = 1 << 1,
 
-  // App names in app.json, including shortName and description
+  /** App names in app.json, including shortName and description */
   ExpoConfigNames = 1 << 2,
 
-  // Android package name in app.json
+  /** Android package name in app.json */
   ExpoConfigAndroidPackage = 1 << 3,
 
-  // iOS bundle identifier in app.json
+  /** iOS bundle identifier in app.json */
   ExpoConfigIosBundleIdentifier = 1 << 4,
 
-  // Schemes in app.json
+  /** Schemes in app.json */
   ExpoConfigSchemes = 1 << 5,
 
-  // EAS project information in app.json
+  /** EAS project information in app.json */
   ExpoConfigEASProject = 1 << 6,
 
-  // Assets in app.json, including icons and splash assets
+  /** Assets in app.json, including icons and splash assets */
   ExpoConfigAssets = 1 << 7,
 
-  // package.json scripts if android and ios items do not contain "run".
-  // Because prebuild will change the scripts in package.json,
-  // this is useful to generate a consistent fingerprint before and after prebuild.
-  PackageJsonAndroidAndIosScriptsIfNotContainRun = 1 << 8,
+  /**
+   * Skip the whole ExpoConfig.
+   * Please the other ExpoConfig source skips when possible and use this flag with caution.
+   * This will potentially skip some native changes.
+   * E.g., add a new config plugin, change app icon, or change the app name.
```

Comment:
```suggestion
   * Prefer the other ExpoConfig source skips when possible and use this flag with caution.
   * This will potentially ignore some native changes that should be part of most fingerprints.
   * E.g., adding a new config plugin, changing the app icon, or changing the app name.
```